From 2f2333c1bc24aa921f2251544188024cbb3e68bb Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 1 Sep 2014 21:18:15 -0400
Subject: [scsi] ipr: increase dump size in ipr driver

Message-id: <1409606300-27001-2-git-send-email-gduarte@redhat.com>
Patchwork-id: 90130
O-Subject: [RHEL7.1 PATCH BZ 1088561 1/6] ipr: increase dump size in ipr driver
Bugzilla: 1088561
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088561
Upstream Status: 95d8a25b52d551f631595ae4b5883d22a8d85a52

commit 95d8a25b52d551f631595ae4b5883d22a8d85a52
Author: wenxiong@linux.vnet.ibm.com <wenxiong@linux.vnet.ibm.com>
Date:   Wed Dec 4 15:22:47 2013 -0600

    [SCSI] ipr: increase dump size in ipr driver

    To help debugging several new SAS controllers, increase the dump size in ipr
    driver to 80MB.

    Signed-off-by: Wen Xiong <wenxiong@linux.vnet.ibm.com>
    Acked-by: Brian King <brking@linux.vnet.ibm.com>
    Signed-off-by: James Bottomley <JBottomley@Parallels.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/ipr.h b/drivers/scsi/ipr.h
index 1c33df2..9ce38a2 100644
--- a/drivers/scsi/ipr.h
+++ b/drivers/scsi/ipr.h
@@ -301,7 +301,7 @@ IPR_PCII_NO_HOST_RRQ | IPR_PCII_IOARRIN_LOST | IPR_PCII_MMIO_ERROR)
  * Dump literals
  */
 #define IPR_FMT2_MAX_IOA_DUMP_SIZE   (4 * 1024 * 1024)
-#define IPR_FMT3_MAX_IOA_DUMP_SIZE   (32 * 1024 * 1024)
+#define IPR_FMT3_MAX_IOA_DUMP_SIZE   (80 * 1024 * 1024)
 #define IPR_FMT2_NUM_SDT_ENTRIES   511
 #define IPR_FMT3_NUM_SDT_ENTRIES   0xFFF
 #define IPR_FMT2_MAX_NUM_DUMP_PAGES ((IPR_FMT2_MAX_IOA_DUMP_SIZE / PAGE_SIZE) + 1)
-- 
1.7.1